megamek/src/megamek/common/AeroSpaceFighter.java

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
We have a standard license notice in https://github.com/MegaMek/megamek/wiki/MegaMek-Coding-Style-Guide